osidb_bindings/__init__.py

```diff
@@ -1,2 +1,3 @@
-from .session import Paginator, new_session
-from .utils import cve_ids
+from .session import Paginator as Paginator
```

Comment:
Why do we need the `as` keyword?

Reply:
To comply with the the ruff warnings https://docs.astral.sh/ruff/rules/unused-import/

Reply:
Could we use the same syntax in all files? I see the `__all__` keyword is used here instead of the `as` https://github.com/RedHatProductSecurity/osidb-bindings/blob/4f6eca5be60601ddb95935d05391b44317129768/osidb_bindings/bindings/python_client/__init__.py#L5-L8

Reply:
Absolutely.